### ISSUE TITLE
Update Linux kernel tree for petitboot

### ISSUE THREAD

**Issue by shenki** [open-power/op-build#236]
As part of the 1.7 release we will track a newer version of Linux, probably 4.1

This is a tracking bug. Please assign it to me

**Comment by williamspatrick** [comment]
@shenki - The current version of Github ACLs doesn't allow me to assign you because you aren't in a group that has write-permission to this repository.

**Comment by williamspatrick** [comment]
Any update?  I'm thinking 1.8 on this one.

**Comment by shenki** [comment]
Yes, let's defer

**Comment by jk-ozlabs** [comment]
We are waiting on an IO driver here, which will likely be submitted against a more recent kernel version than 3.17; this could be good reason to update to a 4.x kernel for v1.7...

**Comment by shenki** [comment]
The following patches are remaining after rebasing on 4.2.

Patches we carry
-----------------------
drm/ast: Default to 8bpp on big endian
xhci: Use xhci_pci_remove for xhci device shutdown
xhci: do not halt the secondary HCD
net/bnx2x: Add shutdown handler

Upstream in 4.3
---------------------
powerpc/kexec: Reset HILE before kexec_sequence
powerpc/kexec: Reset secondary cpu endianess before kexec

Defconfigs
--------------
powerpc: Add openpower_defconfig
powerpc: Add skiroot defconfig

**Comment by stewart-ibm** [comment]
After discussion on drivers and patches needed from them as well as a couple of other open bugs it would be good to have fixed (fixed upstream), we're thinking 4.2 for op-build 1.7 is the best way

**Comment by shenki** [comment]
With PR #264 this has now been merged into master-next. Thanks!
